airbyte-integrations/connectors/source-flexport/unit_tests/test_streams.py

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
Nit: I'd suggest isolating related code in a specific module (`streams.py`) if you use a different test module for streams and sources.